include/circt/Dialect/HW/HWStructure.td

```diff
@@ -129,9 +129,7 @@ def HWModuleOp : HWModuleOpBase<"module",
     void insertOutputs(unsigned index,
                        ArrayRef<std::pair<StringAttr, Value>> outputs);
 
-    // TODO(mlir): FunctionLike shouldn't produce a getBody() helper, it is
-    // squatting on the name.
```

Comment:
Thought this was funny, after 2+ years we can finally remove this TODO since @River707 handled it in https://reviews.llvm.org/D135007.

Reply:
Ha! I laughed at that too :laughing: .
To me that comment suggests we'd prefer to use `getBody()` ourselves but can't because of the squatting-- should we rename this?  Totally fine handling separately / file an issue, just wondering.

Reply:
Yeah, that is now an option. I am honestly not sure if we still care, but I think that can come separately if desired. This TODO was about changing MLIR, which is now done, so I'm going to get rid of it.